Problem when including ZMP controller

@bemilio noticed that:

---
If I indlude the ZMP controller:

`#include <WalkingControllers/SimplifiedModelControllers/ZMPController.h>`
And then I call the method `setFeedback` 
I get the error 

```
/home/emilio/Repositories/robotology-superbuild/build/install/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h:83:42: error: ‘Position’ in namespace ‘iDynTree’ does not name a type
                          const iDynTree::Position& comFeedback);

```

This is solved by adding in `robotology-superbuild/robotology/walking-controllers/src/SimplifiedModelControllers/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h` 

the line: 
```
#include <iDynTree/Core/Position.h>
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,7 +10,7 @@ All notable changes to this project are documented in this file.
 ### Changed
 - Bugfix while resetting the hand smoother in the `RetargetingClient` (https://github.com/robotology/walking-controllers/pull/75)
 - Fixed compilation if iDynTree 3 is used (https://github.com/robotology/walking-controllers/pull/77, https://github.com/robotology/walking-controllers/pull/78).
-
+- Fix missing include in `ZMPController.h` (https://github.com/robotology/walking-controllers/pull/76)
 
 ## [0.4.0] - 2020-12-01
 ### Added

--- a/src/SimplifiedModelControllers/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h
+++ b/src/SimplifiedModelControllers/include/WalkingControllers/SimplifiedModelControllers/ZMPController.h
@@ -21,6 +21,7 @@
 
 // iDynTree
 #include <iDynTree/Core/VectorFixSize.h>
+#include <iDynTree/Core/Position.h>
 
 namespace WalkingControllers
 {

